ComputeDataDrivenFraction.py

```diff
@@ -72,15 +83,23 @@
 legEff.SetY1(0.2)
 legEff.SetY2(0.4)
 
+ptMin = hPromptFrac.GetBinLowEdge(1)
 cFrac = TCanvas('cFrac', '', 800, 800)
-cFrac.DrawFrame(hPromptFrac.GetBinLowEdge(1), 0., ptMax, 1.2, ';#it{p}_{T} (GeV/#it{c}); fraction')
+cFrac.DrawFrame(ptMin, 0., ptMax, 1.2, ';#it{p}_{T} (GeV/#it{c}); fraction')
 hPromptFrac.Draw('same')
 hFDFrac.Draw('same')
 legFrac.Draw()
 cFrac.Update()
 
+cFracNaturalFrac = TCanvas('cFracNaturalFrac', '', 800, 800)
+cFracNaturalFrac.DrawFrame(ptMin, 0., ptMax, 1.2, ';#it{p}_{T} (GeV/#it{c}); corrected fraction')
```

Comment:
Minor comment: in the end, this is the natural fraction of prompt and non-prompt in our dataset, right? So I would write "Natural fraction" on the y axis. Also, the names of the histograms could make this explicit.

Reply:
Yes it is the fraction in "Nature" so without selection biases. I uniformed the names, however I used `Corr` (`corrected`) and not `natural` to be consistent with yields and cross sections (we do not say `natural yields` but `corrected yields`).